src/page/shipping-step.page.ts

```diff
@@ -0,0 +1,17 @@
+
+import { $, ElementFinder } from 'protractor';
+
+export class ShippingStepPage {
+  private checkbox: ElementFinder;
+  private checkoutButton: ElementFinder;
+
+  constructor () {
+    this.checkbox = $('#cgv');
+    this.checkoutButton = $('#form > p > button > span');
+  }
+
+  public async acceptTerms(): Promise<void> {
```

Comment:
as you are performing two actions here, you can name  this method in a better way, like: acceptTermsAndContinue or acceptToSAndCheckout